ui/src/components/Form/DatePicker.vue

```diff
@@ -0,0 +1,75 @@
+<template >
```

Comment:
I don't think this component is necessary. You should update the Date and DateTime components to use this behavior.

Reply:
Is this component necessary now? I think it was incorporated inside the Date and DateTime components if I'm not mistaken so this can probably be deleted.